tests/node-tests.js

```diff
@@ -41,6 +41,42 @@ describe('PapaParse', function() {
 		assertLongSampleParsedCorrectly(Papa.parse(longSampleRawCsv));
 	});
 
+	it('Pause and resume works (Regression Test for Bug #636)', function(done) {
+		this.timeout(30000);
```

Comment:
Why is the timeout required?

Reply:
Default timeout is 2 seconds, the test would take around 8-10 seconds.